x/subspaces/keeper/msg_server.go

```diff
@@ -161,9 +161,12 @@ func (k msgServer) CreateUserGroup(goCtx context.Context, msg *types.MsgCreateUs
 		return nil, sdkerrors.Wrapf(sdkerrors.ErrInvalidAddress, "invalid creator address: %s", msg.Creator)
 	}
 
```

Comment:
Here I will add
```
if !types.IsPermissionValid(msg.DefaultPermissions) {
	return nil, sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, "invalid permission value")
}
```
To make sure that we don't receive an invalid permission value.

Reply:
Done, thanks!

Reply:
Added, thanks!